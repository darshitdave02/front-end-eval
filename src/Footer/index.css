.footer {
  background-color: black;
  color: white;
  display: flex;

  bottom: 0;
  position: sticky;
  font-size: 25px;
  padding-left: 25vw;
  padding-top: 25px;
  padding-bottom: 25px;
  gap: 10px;
}

.purple {
  background-color: #8d0085;
  height: px;
  width: 30px;
  border-radius: 5px;
}

.blue {
  background-color: #200dfe;
  height: px;
  width: 30px;
  border-radius: 5px;
}

.grey {
  background-color: #9b9999;
  height: px;
  width: 30px;
  border-radius: 5px;
}

@media screen and (min-width: 320px) and (max-width: 480px) {
  .footer {
    padding-left: 5vw;
    font-size: 17px;
  }
}
