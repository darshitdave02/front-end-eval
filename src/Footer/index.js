import React from 'react';
import './index.css';
function Footer() {
  return (
    <div className='footer'>
      <div className='footer-title'>THEMES</div>
      <div className='purple'></div>
      <div className='blue'></div>
      <div className='grey'></div>
    </div>
  );
}

export default Footer;
