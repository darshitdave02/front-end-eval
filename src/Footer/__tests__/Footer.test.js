import Footer from '..';

import { render } from '@testing-library/react';
import { useNavigate } from 'react-router-dom';

jest.mock('react-router-dom');

describe('Header', () => {
  it('renders Header component', () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  });
});
