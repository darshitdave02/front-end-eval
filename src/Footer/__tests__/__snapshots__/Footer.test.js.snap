// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders Header component 1`] = `
<DocumentFragment>
  <div
    class="footer"
  >
    <div
      class="footer-title"
    >
      THEMES
    </div>
    <div
      class="purple"
    />
    <div
      class="blue"
    />
    <div
      class="grey"
    />
  </div>
</DocumentFragment>
`;
